src/hotspot/share/gc/shared/cardTable.hpp

```diff
@@ -49,7 +49,12 @@ class CardTable: public CHeapObj<mtGC> {
   CardValue*      _byte_map;         // the card marking array
   CardValue*      _byte_map_base;
 
-  int _cur_covered_regions;
+  // Some barrier sets create tables whose elements correspond to parts of
+  // the heap; the CardTableBarrierSet is an example.  Such barrier sets will
+  // normally reserve space for such tables, and commit parts of the table
+  // "covering" parts of the heap that are committed. At most one covered
+  // region per generation is needed.
+  static constexpr int _max_covered_regions = 2;
```

Comment:
```suggestion
  static constexpr int max_covered_regions = 2;
```
Naming of this static const should follow existing practice in this file (at least ;) )